chromium_src/chrome/browser/notifications/notification_handler.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_NOTIFICATIONS_NOTIFICATION_HANDLER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_NOTIFICATIONS_NOTIFICATION_HANDLER_H_
+
+#define NotificationHandler NotificationHandler_ChromiumImpl
+#include "../../../../../chrome/browser/notifications/notification_handler.h"
+#undef NotificationHandler
+
+class NotificationHandler : public NotificationHandler_ChromiumImpl {
+ public:
+  // GENERATED_JAVA_ENUM_PACKAGE: org.chromium.chrome.browser.notifications
+  // GENERATED_JAVA_CLASS_NAME_OVERRIDE: NotificationType
+  enum class Type {
+    WEB_PERSISTENT = 0,
+    WEB_NON_PERSISTENT = 1,
+    EXTENSION = 2,
+    SEND_TAB_TO_SELF = 3,
+    TRANSIENT = 4,
+    PERMISSION_REQUEST = 5,
+    SHARING = 6,
+    ANNOUNCEMENT = 7,
+    NEARBY_SHARE = 8,
+    BRAVE_ADS = 9,
```

Comment:
Would you apply the following?

```
diff --git a/chromium_src/chrome/browser/notifications/notification_handler.h b/chromium_src/chrome/browser/notifications/notification_handler.h
index 761c28e67b..2941684004 100644
--- a/chromium_src/chrome/browser/notifications/notification_handler.h
+++ b/chromium_src/chrome/browser/notifications/notification_handler.h
@@ -24,7 +24,8 @@ class NotificationHandler : public NotificationHandler_ChromiumImpl {
     SHARING = 6,
     ANNOUNCEMENT = 7,
     NEARBY_SHARE = 8,
-    BRAVE_ADS = 9,
+    NOTIFICATIONS_MUTED = 9,
+    BRAVE_ADS = 10,
     MAX = BRAVE_ADS,
   };
```
Chromium team added `NOTIFICATIONS_MUTED` in October

Reply:
Yes please

Reply:
Pretty sure you'll get build errors without NOTIFICATIONS_MUTED in the enum.

Reply:
Fixed, thanks

Reply:
why are we adding BRAVE_ADS in the middle instead of the end?